addons/im_livechat/static/src/models/thread/thread.js

```diff
@@ -92,6 +97,25 @@ registerInstancePatchModel('mail.thread', 'im_livechat/static/src/models/thread/
     _computeIsChatChannel() {
         return this.channel_type === 'livechat' || this._super();
     },
+    /**
+     * @private
+     * @returns {boolean}
+     */
+    _computeisLiveChatActive() {
+        return (this.channel_type === 'livechat' && this.livechatActive !== undefined) ? this.livechatActive : true;
+    }
```

Comment:
No need for this compute, value should be `undefined` if not channel or not livechat.